packages/utils/telemetry-utils/src/utils.ts

```diff
@@ -2,43 +2,10 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import {
-	ITelemetryBaseEvent,
-	ITelemetryBaseLogger,
-	ITelemetryGenericEvent,
-} from "@fluidframework/core-interfaces";
+import { ITelemetryBaseEvent } from "@fluidframework/core-interfaces";
 import { loggerToMonitoringContext } from "./config";
 import { ITelemetryGenericEventExt, ITelemetryLoggerExt } from "./telemetryTypes";
 
-/**
- * Like assert, but logs only if the condition is false, rather than throwing
- * @param condition - The condition to attest too
- * @param logger - The logger to log with
- * @param event - The string or event to log
- * @returns The outcome of the condition
- *
- * @internal
```

Comment:
Why are we removing it?  It's a useful `@internal` helper. Right now it looks like there's only one place that would use it (`deltaManager` has its own copy), but when the next bit of code wants this, shouldn't we have it ready in a shared place?

Reply:
I would argue for re-adding it (or something like it) as needed. Since we only have 1 use of this pattern in the repo, it seems like this pattern isn't widely needed currently.

Reply:
Yeah, we deliberately copied it to deltaManager and deprecated this because it was the only use case. As Josh says, happy to get it somewhere shared when we actually need it.

Reply:
ok